src/CSAccounting.sol

```diff
@@ -574,26 +577,30 @@ contract CSAccounting is
         uint256 nodeOperatorId,
         uint256 nonWithdrawnKeys
     ) internal view returns (uint256 current, uint256 required) {
-        current = CSBondCore.getBond(nodeOperatorId);
-        required =
-            CSBondCurve.getBondAmountByKeysCount(
-                nonWithdrawnKeys,
-                CSBondCurve.getBondCurve(nodeOperatorId)
-            ) +
-            CSBondLock.getActualLockedBond(nodeOperatorId);
+        unchecked {
+            current = CSBondCore.getBond(nodeOperatorId);
+            required =
+                CSBondCurve.getBondAmountByKeysCount(
+                    nonWithdrawnKeys,
+                    CSBondCurve.getBondCurve(nodeOperatorId)
+                ) +
+                CSBondLock.getActualLockedBond(nodeOperatorId);
```

Comment:
Add comment that `getActualLockedBond` is uint128, so no overflow expected in practice.